Homework 3/style.css

```diff
@@ -127,3 +138,81 @@ textarea{
 .wrapword{
 	white-space: pre-wrap;
 }
+.deleteButton{
+	width:40px;
+	height:40px;
+	font-size: 25px;
+	color: white;
+	background-color: rgba(0, 0, 0, 0);
+	padding: 0px;
+	margin: 0px;
+}
+.addButton{
+	width:40px;
+	height:40px;
+	font-size: 25px;
+	color: white;
+	background-color: rgba(0, 0, 0, 0);
+	padding:0px;
+	margin: 0px;
+}
+.buttonText{
+	margin-top: -20px;
+}
+#ownerStoreImage{
+	max-height: 200px;
+}
+#ownerStoreTitle{
+	float:left;
+	margin-left: 50px;
+	margin-right: 50px;
+}
+#ownerStoreProducts{
```

Comment:
You're duplicating css right now. You can just make a class for similar styles and then make an ID to overwrite specifics.